ingester/ingester.go

```diff
@@ -570,12 +579,23 @@ func (i *Ingester) flushLoop(j int) {
 		}
 
 		// flush the chunks without locking the series
-		if err := i.flushChunks(ctx, op.fp, series.metric, chunks); err != nil {
+		err := i.flushChunks(ctx, op.fp, series.metric, chunks)
+		if err != nil {
+			if awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == provisionedThroughputExceededException {
```

Comment:
Instrumenting here would help fix https://github.com/weaveworks/monitoring/issues/12.

(No change required, immediately. One of us will get to it soon enough.)